services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowValidationNegativeSpec.groovy

```diff
@@ -117,6 +118,24 @@ class FlowValidationNegativeSpec extends BaseSpecification {
         "synchronize" | "Could not reroute flow: Flow $NON_EXISTENT_FLOW_ID not found"
     }
 
+    def "Unable to create a flow on an isl port"() {
+        given: "A isl"
+        Isl isl = topology.islsForActiveSwitches.find { it.aswitch && it.dstSwitch }
+        assert isl
+
+        when: "Try to create a flow using isl port"
+        def flow = flowHelper.randomFlow(isl.srcSwitch, isl.dstSwitch)
+        flow.source.portNumber = isl.srcPort
+        flow.destination.portNumber = isl.dstPort
```

Comment:
Please make 2 iterations. 1 for src violation and 1 for dst violation

Reply:
done.